terraform/modules/hub/files/tasks/frontend.json

```diff
@@ -99,6 +102,9 @@
     }, {
       "Name": "AB_TEST_FILE",
       "Value": "/verify-frontend/federation/configuration/ab_test/ab_test.yml"
+    }, {
+      "Name": "AB_TEST_FILE",
```

Comment:
no need to duplicate it, see above.

Reply:
I deleted that before committing and pushing.  I just forgot to add the changed file before committing and pushing 😄